### DISCUSSION TITLE
include blade component in placeholder field (without creating custom field)?

### DISCUSSION THREAD

**Discussion by wivaku** [filamentphp/filament#8106] [Help] [answered]
I can create a custom field for a badge in a placeholder field (https://github.com/filamentphp/filament/discussions/6563).

Sometimes creating a custom field is overkill.
How can I achieve the same thing without a custom field?

I.e. use a [Badge Blade Component](https://filamentphp.com/docs/3.x/support/blade-components/badge) directly in the Placeholder field. 
 
The following does not work, as it inserts the literal string in HTML:
```
Forms\Components\Placeholder::make('status')
    ->content(fn($record) => new HtmlString("<x-filament::badge>{$record->status}</x-filament::badge>")),
```

Result:
```
<div class="fi-fo-placeholder sm:text-sm">
    <x-filament::badge>processed</x-filament::badge>
</div>
```

**Comment by danharrin** [comment, marked as answer]
`fn($record) => new HtmlString(Blade::render("<x-filament::badge>{$record->status}</x-filament::badge>"))`

Still, I recommend a custom field. I find HTML in .php classes ugly.

**Reply by wivaku** [reply]
thanks for the answer. And yes, it's ugly. 😄
